MSVC build for appveyor is broken

```
error[E0308]: mismatched types
  --> C:\projects\servo\components\gfx\font_template.rs:83:29
   |
83 |             Some(_) => Some(try!(FontTemplateData::new(identifier.clone(), maybe_bytes))),
   |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `platform::dummy::font_template::FontTemplateData`, found enum `std::result::Result`
   |
   = note: expected type `platform::dummy::font_template::FontTemplateData`
   = note:    found type `std::result::Result<_, _>`
   = note: this error originates in a macro from the standard library
error[E0308]: mismatched types
  --> C:\projects\servo\components\gfx\font_template.rs:83:29
   |
83 |             Some(_) => Some(try!(FontTemplateData::new(identifier.clone(), maybe_bytes))),
   |                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `platform::dummy::font_template::FontTemplateData`, found enum `std::result::Result`
   |
   = note: expected type `platform::dummy::font_template::FontTemplateData`
   = note:    found type `std::result::Result<_, _>`
   = note: this error originates in a macro from the standard library
error[E0308]: mismatched types
   --> C:\projects\servo\components\gfx\font_template.rs:205:38
    |
205 |         let template_data = Arc::new(try!(FontTemplateData::new(self.identifier.clone(), None)));
    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `platform::dummy::font_template::FontTemplateData`, found enum `std::result::Result`
    |
    = note: expected type `platform::dummy::font_template::FontTemplateData`
    = note:    found type `std::result::Result<_, _>`
    = note: this error originates in a macro from the standard library
error[E0308]: mismatched types
   --> C:\projects\servo\components\gfx\font_template.rs:205:38
    |
205 |         let template_data = Arc::new(try!(FontTemplateData::new(self.identifier.clone(), None)));
    |                                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected struct `platform::dummy::font_template::FontTemplateData`, found enum `std::result::Result`
    |
    = note: expected type `platform::dummy::font_template::FontTemplateData`
    = note:    found type `std::result::Result<_, _>`
    = note: this error originates in a macro from the standard library
error: aborting due to 4 previous errors
Build failed, waiting for other jobs to finish...
error: Could not compile `gfx`.
```

--- a/components/gfx/platform/dummy/font_template.rs
+++ b/components/gfx/platform/dummy/font_template.rs
@@ -2,6 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+use std::io::Error;
 use string_cache::Atom;
 use webrender_traits::NativeFontHandle;
 
@@ -12,7 +13,7 @@ pub struct FontTemplateData {
 }
 
 impl FontTemplateData {
-    pub fn new(identifier: Atom, font_data: Option<Vec<u8>>) -> FontTemplateData {
+    pub fn new(identifier: Atom, font_data: Option<Vec<u8>>) -> Result<FontTemplateData, Error> {
         let bytes = match font_data {
             Some(bytes) => {
                 bytes
@@ -22,10 +23,10 @@ impl FontTemplateData {
             }
         };
 
-        FontTemplateData {
+        Ok(FontTemplateData {
             bytes: bytes,
             identifier: identifier,
-        }
+        })
     }
     pub fn bytes(&self) -> Vec<u8> {
         self.bytes.clone()

